dll/win32/comdlg32/filedlg.c

```diff
@@ -336,7 +336,12 @@ static BOOL get_config_key_string(HKEY hkey, const WCHAR *name, WCHAR **value)
     DWORD type, size;
     WCHAR *str;
 
-    if (hkey && !RegQueryValueExW(hkey, name, 0, &type, NULL, &size))
+    if (!hkey)
+    {
+        return FALSE;
+    }
+
+    if (!RegQueryValueExW(hkey, name, 0, &type, NULL, &size))
```

Comment:
If this is code change in Wine code: Surround the modification in `#ifdef __REACTOS__  ... #endif`